app/mailers/user_mailer.rb

```diff
@@ -16,6 +18,17 @@ def notification(title: '', body: '', path: '', emails: [])
          to: emails
   end
 
+  def async_report_notification(email:, start_time:, completion_time:, record_count:, url:)
```

Comment:
Ryan, you mentioned making some readability improvements to the email. If you're doing that, I think it would be useful to include the name of the exported file in the email body and/or subject, unless there's some security reason not to.

Reply:
good point. i've noted this in Trello and we can follow up with feature